src/main/java/com/nha/abdm/wrapper/hiu/facade/consent/HIUFacadeConsentController.java

```diff
@@ -0,0 +1,43 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hiu.facade.consent;
+
+import com.nha.abdm.wrapper.common.responses.FacadeResponse;
+import com.nha.abdm.wrapper.hiu.hrp.consent.HIUConsentInterface;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.ConsentStatusRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.FetchConsentRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.InitConsentRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.OnNotifyRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HIUFacadeConsentController {
```

Comment:
I am assuming this classes exposes APIs to be called by the HIU. Does this class makes the APIs synchronous, can you please document these?

Reply:
Added an issue to handle this: https://github.com/NHA-ABDM/ABDM-wrapper/issues/45